### PR TITLE
Document the Switch component

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

File: docs/src/routes/docs/components/switch.svx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+| **`inputClass`** | `null` | `string` | A class string to add to the `<input>` inside. |
+| **`trackClass`** | `null` | `string` | A class string to add to the [track](https://material.io/components/selection-controls#switches) of the switch. |
+| **`thumbClass`** | `null` | `string` | A class string to add to the [thumb](https://material.io/components/selection-controls#switches) of the switch. |
+
+
+## Events {#events}
+
+| Name | Event Detail | Description |
+| ---- | ------------ | ----------- |
+| **`event`** | `{ value, nativeEvent }` | Fires when a switch is toggled. The `value` corresponds to the current value of the ==`value`== property, the `nativeEvent` contains the [native change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event). |
```
Comment (aabounegm) [inline docs/src/routes/docs/components/switch.svx:63]: I'm pretty sure the event is called `change`, not `event`

File: docs/src/routes/docs/components/switch.svx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+## Slots {#slots}
+
+### Default slot
+
+The label of the switch. Positioned to the right of the switch by default, can be moved to the left using the ==`slotLeft`== property.
```
Comment (aabounegm) [inline docs/src/routes/docs/components/switch.svx:70]: Can't help but notice the comma splice here. (yes, I can sometimes be much of a grammar nazi)

Comment (illright) [reply docs/src/routes/docs/components/switch.svx:70]: I had to google what a comma splice is) okay, will fix

**Comment by illright** [comment]
We might not be running out of space, but I ran out of imagination, so no slot left)

**Pull request merged**